tests/src/helpers.c

```diff
@@ -244,3 +244,67 @@ void mbedtls_param_failed( const char *failure_condition,
     }
 }
 #endif /* MBEDTLS_CHECK_PARAMS */
+
+#if defined(MBEDTLS_PSA_CRYPTO_BUILTIN_KEYS)
+
+#include <psa/crypto.h>
+
+typedef struct
+{
+    psa_key_location_t location;
+    psa_drv_slot_number_t slot_number;
+} mbedtls_psa_builtin_key_description_t;
+
+static const mbedtls_psa_builtin_key_description_t builtin_keys[] = {
+    // TODO: declare some keys
+    {0, 0},
+};
+
+psa_status_t mbedtls_psa_platform_get_builtin_key(
```

Comment:
If this is going to be autogenerated code, wouldn't it make sense to have it inside the `_wrappers.c` file?